Checkers/Checkers.cs

```diff
@@ -8,79 +8,256 @@ class Program
         {
-            // Your code here
-            return;
+            //this method actually moves the chekers
+            Console.WriteLine("Enter Pickup Row:");
+            int x1 = Convert.ToInt32(Console.ReadLine());
+            Console.WriteLine("Enter Pickup Column:");
+            int y1 = Convert.ToInt32(Console.ReadLine());
+            //x1 and y1 are the original spot in that turn for the checkers to be moved
+
+            Console.WriteLine("Enter Placement Row:");
+            int x2 = Convert.ToInt32(Console.ReadLine());
+            Console.WriteLine("Enter Placement Column");
+            int y2 = Convert.ToInt32(Console.ReadLine());
+            //x2 and y2 are the coordinates the checker will be moved to
+            Checker check;
+            try 
+            {
+                check = SelectChecker(x1,y1);
+                //uses the first two coordinates from the user in this turn
+                //to see if there is a checker there that can be moved
+            }
+            catch 
+            {
+                Console.WriteLine("There is no checker in that spot.");
+                return;
+            }
+
+
+            int WorB = 0;
+            //this integer helps us change the math easily for white or black checkers moving on the grid
+            if(check.Color == "white")
+            {
+                WorB = 1;
+            }
+            else if (check.Color == "black")
+            {
+                WorB = -1;
+            }
+
+            if ((x2 <= 7) && (x2 >= 0) && (y2 <= 7) && (y2 >= 0)) 
+            //first of all, the move needs to be within the parameters of our board
+            {
+                //on the board
+                if ((y2 == y1+1) || (y2 == y1-1) && (x2 == x1+WorB))
+                {
+                    //one way diagonal spot
+                    Checker newSpace;
+
+                        //see if we find a checker in the target spot
+                        newSpace = SelectChecker(x2,y2);
+
+                        Console.WriteLine("no checker found in space");
+                        //if no checker present, move 1 diagonally
+
+                        check.Position[0] = x2;
+                        check.Position[1] = y2;
+
+                        DrawBoard();
+                        //redraw pur board to re-place checkers
+
+                    //if a cheker is found in the spot
+                    if(newSpace != null) 
+                    {
+                        if (check.Color == newSpace.Color)
+                        {
+                            Console.WriteLine("That spot is a friendly checker, You can't move there.");
+                            return;
+                        }
+                        else if (check.Color != newSpace.Color)
+                        //your move to coordinates need to be ON the enemy checker in order to jump
+                        //but you do end up in the right spot
+                        {
```

Comment:
You are checking that you are capturing a checker, but not that the spot beyond it is open